docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -132,6 +132,11 @@ Noteworthy plugin changes
 * Cache plugins in collections can be used to cache data from inventory plugins. Previously, cache plugins in collections could only be used for fact caching.
 * Some undocumented arguments from ``FILE_COMMON_ARGUMENTS`` have been removed; plugins using these, in particular action plugins, need to be adjusted. The undocumented arguments which were removed are ``src``, ``follow``, ``force``, ``content``, ``backup``, ``remote_src``, ``regexp``, ``delimiter``, and ``directory_mode``.
 
+Action module calls should use FQ names
```

Comment:
I'm guessing the title is just misleading for me, but it sounds like you are referring to `actions` as `modules`, but I'm guessing this is meant to be "Action plugins should call modules or other actions using FQ names"

Reply:
Good point- Rick wordsmithed a clearer suggestion that I've added